### PR TITLE
Fill-out chapter 1

### PR DESCRIPTION
**Pull request by latazzajones**
Just added some more info in spots and cleaned up some spacing. I also added `$` in front of all the commands and a screenshot.

### CODE REVIEW

File: _chapters/chapter_01.md [hunk 1]
```diff
@@ -72,13 +80,21 @@ layout: rails_tutorial
 {% endhighlight %}
 {% endsteps %}
 
+{% aside %}
+  The command "bundle" might sound familiar to you. If you did the install instructions for this tutorial, you probably ran `$gem install bundler`. 
+
+  [Bundler](http://bundler.io/) is a program that helps you manage all of the other little programs that help Rails run and all of the commands related to that program start with `$bundle`. 
+
+  You don't need to worry about this much for the tutorial but if you're interested in it, you can learn more about them on their website. 
+{% endaside %}
```
Comment (alimi) [inline _chapters/chapter_01.md:91]: Is this extra info about Bundler necessary? This is the only time Bundler gets used throughout the entire tutorial, and it gets used indirectly.

Bundler is really awesome, but without more context in the tutorial I'm not sure we gain much by adding this little detail.

File: _chapters/chapter_01.md [hunk 2]
```diff
@@ -72,13 +80,21 @@ layout: rails_tutorial
...
+
 {% steps %}
 {% list %}
-  1.  Let's take a look the files that got created.
+  1.  Let's take a look the files that got created when you ran `$rails new`.
 
-  1.  Run `ls -l` to list out all the top level files and directories `rails new` generated. There is lot going on, but a few things stand out.
+  1.  Run `ls -l` to list out all the top level files and directories that you created. There's lot going on, but a few things stand out.
```
Comment (alimi) [inline _chapters/chapter_01.md]: that you created...by running `rails new`?

File: _chapters/chapter_01.md [hunk 3]
```diff
@@ -72,13 +80,21 @@ layout: rails_tutorial
...
 
-  1.  Any code that is specific to the bookstore application will live in the `app` directory. Run `ls -l app` to take a look at the some of the things you'll be working with throughout the tutorial.
+  1.  Any code that is specific to the bookstore application will live in the `app` directory this is where you'll be doing most of your work for this tutorial. Run `ls -l app` to take a look at the some of the things you'll be working with throughout the tutorial.
```
Comment (alimi) [inline _chapters/chapter_01.md]: This is a run on sentence.

File: _chapters/chapter_01.md [hunk 4]
```diff
@@ -114,22 +130,22 @@ layout: rails_tutorial
 {% endsteps %}
 
 {% protip %}
-  Don't have a favorite text editor for programming? We recommend [Atom](https://atom.io/).
+  It's easiest to edit these files using a text editor created just for programming. If you don't already have a favorite text editor for programming? We recommend [Atom](https://atom.io/).
```
Comment (alimi) [inline _chapters/chapter_01.md]: Nice! I hard time wording this one. The last couple of sentences are kinda funny though. Maybe you forgot to finish your thought?

>If you don't already have a favorite text editor for programming? We recommend [Atom](https://atom.io/).

File: _chapters/chapter_01.md [hunk 5]
```diff
@@ -114,22 +130,22 @@ layout: rails_tutorial
...
 {% endprotip %}
 
 {% steps %}
 {% list %}
-  1.  The `config` directory is where you'll store configuration files for the bookstore application. `rails new` generated a bunch of configuration files to help you get this new application up and running quickly. Let's take a look at one of those files.
+  1.  The `config` directory is where you'll store configuration files for the bookstore application. `$rails new` generated a bunch of configuration files to help you get this new application up and running quickly. Let's take a look at one of those files.
 
-  1.  Use your favorite text editor to open the `bookstore` directory.
+  1.  From inside the menu of your favorite text editor to open the whole `bookstore` directory.
```
Comment (alimi) [inline _chapters/chapter_01.md]: The wording here feels really awkward.

File: _chapters/chapter_01.md [hunk 6]
```diff
@@ -114,22 +130,22 @@ layout: rails_tutorial
...
 
-  1.  In your text editor, open the `config` directory, and then open `database.yml`.
+  1.  In your text editor, navigate to the `config` directory, and then look at the contents of the `database.yml` file.
 
   1.  `database.yml` tells Rails how to connect to the bookstore application's database.
 
   1.  By default, the bookstore application is setup to use [SQLite](https://sqlite.org/). It's a light weight database that's perfect for this tutorial.
 
-  1.  Since we're happy with the default `database.yml` generated by `rails new`, we can live it as is. Ah, [convention over configuration](https://en.wikipedia.org/wiki/Convention_over_configuration).
+  1.  Since we're happy with the default `database.yml` generated by `rails new`, we can live it as is. Which is great because we ❤️ [convention over configuration](https://en.wikipedia.org/wiki/Convention_over_configuration) when programming with Rails.
```
Comment (alimi) [inline _chapters/chapter_01.md:150]: Surprisingly, I like my voice better 😏

File: _chapters/chapter_01.md [hunk 7]
```diff
@@ -166,24 +182,31 @@ layout: rails_tutorial
 {% aside %}
   Not sure what a database is? Don't worry, we'll fill you in :)
 
-  Brief description with external resources if we think this is necessary.
+  There are many different types of databases but the one we'll be working with here - and that's the most similar to what you've probably worked with in the past is called a Relational Database. There are many types of *Relational Database Management Systems* (RDMS's for short) and SQLite is just one of them. 
+
+Feel free to check out this video to learn a bit more about Relational Databases: 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/t48TGntrX4s" frameborder="0" allowfullscreen></iframe>
```
Comment (alimi) [inline _chapters/chapter_01.md:191]: I *think* I'd just link out to the video.

File: _chapters/chapter_01.md [hunk 8]
```diff
@@ -166,24 +182,31 @@ layout: rails_tutorial
 {% aside %}
   Not sure what a database is? Don't worry, we'll fill you in :)
 
-  Brief description with external resources if we think this is necessary.
+  There are many different types of databases but the one we'll be working with here - and that's the most similar to what you've probably worked with in the past is called a Relational Database. There are many types of *Relational Database Management Systems* (RDMS's for short) and SQLite is just one of them.
```
Comment (alimi) [inline _chapters/chapter_01.md]: This is good, but would it be better to have a brief description of databases? I'm not sure what that description would be, but maybe something like "A database is made up of tables where you store stuff...kinda like an Excel spreadsheet"?

Reading the description we have now doesn't really answer the question of "What is a database".

File: _chapters/chapter_01.md [hunk 9]
```diff
@@ -166,24 +182,31 @@ layout: rails_tutorial
...
+
+Feel free to check out this video to learn a bit more about Relational Databases: 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/t48TGntrX4s" frameborder="0" allowfullscreen></iframe>
+
 {% endaside %}
 
 {% protip %}
- Don't worry if what you're seeing doesn't look exactly like the examples we've included in the tutorial. Small things can change between different versions of Rails.
+ As you work through the tutorial - don't worry if what you're seeing doesn't look exactly like the examples we've included in the tutorial. Small things can change between different versions of Rails.
```
Comment (alimi) [inline _chapters/chapter_01.md:196]: This *feels* a little too wordy.

File: _chapters/chapter_01.md [hunk 10]
```diff
@@ -238,11 +261,14 @@ layout: rails_tutorial
 {% aside %}
   What's this bookstore we keep talking about? In this tutorial, we're going to use Rails to make an application for a bookstore.
 
-  As you build the application, you'll add features to create, view, edit, and delete books. Since we all have feelings on the books we love...and hate, you'll add features so books can be reviewed by users.
+  As you build the application, you'll add features to `create`, `view`, `edit`, and `delete` books. Since we all have feelings on the books we love...and hate, you'll add features so books can be reviewed by users.
```
Comment (alimi) [inline _chapters/chapter_01.md:266]: I think create, view, etc. should be treated as words, not code. I was trying to translate the coding stuff into people talk.

File: _chapters/chapter_01.md [hunk 11]
```diff
@@ -9,22 +9,30 @@ layout: rails_tutorial
...
+
+  * To run this command on Windows you'll need to type it into the *Command Prompt* then press `enter`. 
+  
+  Throughout this tutorial we'll be referring to both Terminal and Command Prompt as "Terminal". 
 {% endprotip%}
 
 {% steps %}
 {% list %}
-  1.  Open Terminal.
+  1.  Get started by opening your Terminal.
```
Comment (alimi) [inline _chapters/chapter_01.md]: I've been writing "the" Terminal instead of "your" Terminal throughout the tutorial. I *think* I like "the Terminal" better, but I can be convinced otherwise.

File: _chapters/chapter_01.md [hunk 12]
```diff
@@ -9,22 +9,30 @@ layout: rails_tutorial
...
-  1.  You're now in your home directory. Run `ls` to see what's in your home directory. You might see some familiar directories like `Desktop`, `Documents`, and `Music`.
+  1.  You're now in your home directory. The command `ls` will always show you what's in the current directory. Go ahead and run `ls` to see what files are here.
+ 
+  1.  You might see some familiar files like Desktop, Documents, and Music.
 
-  1.  All code deserves a good home. Add a new `Projects` directory to your home directory by running `mkdir Projects`.
+  1.  All code deserves a good home. Add a new `Projects` directory to your home directory by running `$mkdir Projects`.
 
-  1.  Navigate to your new `Projects` directory by running `cd Projects`.
+  1.  Navigate into your new `Projects` directory by running `$cd Projects`.
```
Comment (alimi) [inline _chapters/chapter_01.md]: Good call.

File: _chapters/chapter_01.md [hunk 13]
```diff
@@ -40,16 +48,16 @@ layout: rails_tutorial
 {% endsteps %}
 
 {% protip %}
-  Before you can complete the following list, your computer needs to be configured with Rails. If you haven't already, please refer to the [install instructions]({{ site.baseurl }}/installation.html) for list on configuring your computer.
+  Before you can continue with the tutorial, your computer needs to be configured with Rails. If you haven't already, please refer to the [install instructions]({{ site.baseurl }}/installation.html) to get everything set up and ready to go.
```
Comment (alimi) [inline _chapters/chapter_01.md:51]: 👏🏾

File: _chapters/chapter_01.md [hunk 14]
```diff
@@ -9,22 +9,30 @@ layout: rails_tutorial
 {% endsectionheader %}
 
 {% protip %}
-  Throughout the tutorial, you're going to run a lot of Terminal commands. For example, we might ask you to do something like
+  Throughout the tutorial, you're going to run a lot of commands on the command line. For example, we might ask you to do something like
 
-  $ Run `pwd` to print your current working directory.
+  > Run `$pwd` to print your current working directory.
```
Comment (alimi) [inline _chapters/chapter_01.md]: `$pwd` looks good in the browser, but it might be confusing. You're going to type `pwd` in the Terminal, not `$pwd`.

**Review by alimi — COMMENTED** [review]
Here's my feedback ❤️

File: _chapters/chapter_01.md (hunk 2 above)
Comment (latazzajones) [reply _chapters/chapter_01.md]: ya - i just took it out because it's redundant. the point above it sets up that we're talking about that ... and it sounds weird to have the phrase rails new over and over.

Comment (latazzajones) [reply _chapters/chapter_01.md]: n/m I made a better edit

File: _chapters/chapter_01.md (hunk 6 above)
Comment (latazzajones) [reply _chapters/chapter_01.md:150]: tough
